tests/test_filter_panels.py

```diff
@@ -0,0 +1,91 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_edit_filter_panels(base_url, selenium):
+    """Open Treeherder page, open Filters Panel, select one filter,
+    verify results"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    # Get next 50 jobs to see 'exception' unclassified failures
+    page.get_next_fifty_results()
+    page.deselect_all_failures()
+
+    # Check 'testfailed' unclassified failures
+    page.select_testfailed_failures()
+    page.open_next_unclassified_failure()
+    teststatus = page.job_details.job_result_status
+    assert teststatus in ['testfailed']
+    page.select_testfailed_failures()
```

Comment:
It would read better if we had separate methods for selecting/deselecting by failure status. This also give us the benefit of checking the initial state. We can also manage this in a page region for the filters, to avoid overloading the page object. For example the test might look like:

```python
filters = page.open_filters()
filters.failures.hide_all()  # hide all failures
filters.failures.show_testfailed()  # show testfailed failures
page.open_next_unclassified_failure()
assert 'testfailed' == page.job_details.job_result_status
filters.failures.hide_testfailed()
...
```